draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -204,35 +204,33 @@ As a simplification, some applications may decide to omit the "-challenge" suffi
 
 ## Time-bound checking and Expiration
 
-After domain control validation is completed, there is typically no need for the TXT or CNAME record to continue to exist as the presence of the domain validation DNS record for a service only implies that a User with access to the service also has DNS control of the domain at the time the code was generated. It should be safe to remove the validation DNS record once the validation is done and the Application Service Provider doing the validation should specify how long the validation will take (i.e., after how much time can the validation DNS record be deleted).
-
-Some Application Service Providers currently require the Validation Record to remain in the zone indefinitely for periodic revalidation purposes. This practice should be discouraged. Subsequent validation actions using an already disclosed token are no guarantee that the original owner is still in control of the domain, and a new challenge needs to be issued.
```

Comment:
this should not just be removed but changed. eg like "Some Application Service Providers currently re-use the Validation Record as a continued confirmation of wanting to consume a previously activated service" [...]

Reply:
I agree we should change it rather than remove it, but this should perhaps be a separate PR.

Reply:
I believe Eric has added this back in.